src/python/module/smartredis/client.py

```diff
@@ -121,7 +121,7 @@ def __deprecated_construction(self, cluster, address=None, logger_name="Default"
         except (PybindRedisReplyError, RuntimeError) as e:
             raise RedisConnectionError(str(e)) from None
 
-    def __new_construction(self, config_options=None, logger_name="Default"):  # pylint: disable=no-self-use
+    def __standard_construction(self, config_options=None, logger_name="Default"):  # pylint: disable=no-self-use
```

Comment:
Again, not desperately important, but if we could we retroactively type hint this while we are here?
```py
    def __standard_construction(
        self,
        config_options t.Optional[ConfigOptions] = None,
        logger_name: str = "Default"
    ) -> PyClient:
```

Reply:
Done